### DISCUSSION TITLE
Can GitVersion Be Used Via Git Fork WorkFlow

### DISCUSSION THREAD

**Discussion by sportyturks** [GitTools/GitVersion#3670] [Q&A] [answered]
New to GitVersion and in the process of doing some investigation and was wondering if GitVersion can be used when following a Git Fork Workflow branching strategy or is it strictly for a Git Flow/GitHub Flow branching strategy only? 

Is there something that I need to consider when using GitVersion with the Git Fork Workflow branching strategy?

**Comment by HHobeck** [comment, marked as answer]
Thank you for giving GitVersion a try. Git version supports natively GitFlow and GitHubFlow. In the current beta release version (6.0.0-beta.3) we have a new feature integrated to inherit from supported workflows like:

```yaml
workspace: GitFlow/v1
next-version: 1.0.0
```
https://github.com/GitTools/GitVersion/tree/main/src/GitVersion.Core/Configuration/SupportedWorkflows
https://gitversion.net/docs/learn/branching-strategies/gitflow/examples

I don't know the Git Fork workflow but git version is quite generic and can be configured to support custom workflows. If you think the Git Fork is a common standard and should be provided to other users as a base template then you are welcome creating a pull-request.
